stack/base/Dockerfile

```diff
@@ -44,11 +44,10 @@ ENV PIP_CONSTRAINT /opt/requirements.txt
 # Ensure that pip installs packages to ~/.local by default
 ENV PIP_USER 1
 
-# Upgrade pip to latest
-RUN mamba update -y pip && mamba clean --all -f -y
-
+# Upgrade pip and mamba to latest
 # Install aiida-core and other shared requirements.
-RUN mamba install --yes \
+RUN mamba update -y pip && \
```

Comment:
Combining `mamba update` and `mamba install` in a single RUN should speed up the build slightly since we need to download the conda-forge package index only once.